include/aspect/simulator.h

```diff
@@ -804,7 +804,8 @@ namespace aspect
        */
       void denormalize_pressure(LinearAlgebra::BlockVector &vector);
 
-
+//FIXME apply the limiter to dg solutions
```

Comment:
Oh, and please make a comment about the purpose of the function in the usual doxygen notation of the other comments (using /** ... **/)